### ISSUE TITLE
why not use $digest instead of $apply

### ISSUE THREAD

**Issue by mato75** [kamilkp/angular-vs-repeat#60]
Why not use $digest in scroll, it is faster then $apply, but you cannot use the new setting latch, because that slows it down.

           if (updateInnerCollection()) {
                $scope.$digest();
                // $scope.$apply();
                $scope.$broadcast('vsSetOffset-refresh');
          }

**Comment by jramoyo** [comment]
I agree. For large applications, the scroll performance degrades significantly when using $scope.$apply().

From AngularJS Batarang:

$scope.$apply()
![apply](https://cloud.githubusercontent.com/assets/883118/11527212/81968ee6-991a-11e5-941d-5ac5221f7004.png)

$scope.$digest()
![digest](https://cloud.githubusercontent.com/assets/883118/11527211/8196273a-991a-11e5-9eba-644382a84277.png)
